bindings/matrix-sdk-ffi/src/session_verification.rs

```diff
@@ -53,8 +54,11 @@ pub struct SessionVerificationController {
 
 #[uniffi::export(async_runtime = "tokio")]
 impl SessionVerificationController {
-    pub fn is_verified(&self) -> bool {
-        self.user_identity.is_verified()
+    pub async fn is_verified(&self) -> Result<bool, ClientError> {
+        let device =
+            self.encryption.get_own_device().await?.context("Failed retrieving own device")?;
```

Comment:
That's `Option::context` we're calling here (the failure would be the `?` after the `await`), so let's make the message slightly more precise:

```suggestion
            self.encryption.get_own_device().await?.context("Our own device is missing")?;
```